### PR TITLE
Announcement for Jansa Alpha release

### PR DESCRIPTION
**Pull request by chessbyte**

### CODE REVIEW

File: site/_posts/2020-03-31-Announcing-Jansa-Alpha.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+We have just built Jansa Alpha. This release has 2 major changes that are included in the [roadmap](https://github.com/orgs/ManageIQ/projects/13#column-7337345)
+
+- Podification
+- VMware VimBroker Removal - described [here](http://talk.manageiq.org/t/vmware-vimbroker-removal/4775)
+
+Since this is an alpha release, we do expect some period of stabilization before we release the GA version.  Please try it and let us know of any issues you encounter to help us squash any remaining bugs.
+
+You can download the Jansa Alpha (jansa-1-alpha) release [here](http://manageiq.org/download/).
```
Comment (Fryguy) [inline site/_posts/2020-03-31-Announcing-Jansa-Alpha.md]: this should be `jansa-1-alpha1` in the paresn.

File: site/_posts/2020-03-31-Announcing-Jansa-Alpha.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+author: chessbyte
+date: 2020-03-31 19:12:13 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+We have just built Jansa Alpha. This release has 2 major changes that are included in the [roadmap](https://github.com/orgs/ManageIQ/projects/13#column-7337345)
+
+- Podification
```
Comment (Fryguy) [inline site/_posts/2020-03-31-Announcing-Jansa-Alpha.md]: Perhaps something like:

- Podification - A new release format for running ManageIQ on Kubernetes and OpenShift.  This is the culmination of the rearchitecture we started almost 2 years ago.  While we've had a "podified" version for some time, it was structurally different and not really supported nor advertised, being more of a stepping stone to this new format.

File: site/_posts/2020-03-31-Announcing-Jansa-Alpha.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- VMware VimBroker Removal - described [here](http://talk.manageiq.org/t/vmware-vimbroker-removal/4775)
```
Comment (Fryguy) [inline site/_posts/2020-03-31-Announcing-Jansa-Alpha.md]: - VMware VimBroker Removal - In addition to the various performance issues with the VMware VimBroker, one of the underlying technologies, [DRb](https://en.wikipedia.org/wiki/Distributed_Ruby) was incompatible with Podification.  As such, we redesigned the VMware interaction in such a way that the broker was removed completely.  Much more detail can be found [here](http://talk.manageiq.org/t/vmware-vimbroker-removal/4775).

File: site/_posts/2020-03-31-Announcing-Jansa-Alpha.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- Podification - A new release format for running ManageIQ on Kubernetes and OpenShift. This is the culmination of the [rearchitecture](https://github.com/ManageIQ/manageiq-design/issues/32) we started in 2017. While we've had a "podified" version for some time, it was structurally different and not really supported nor advertised, being more of a stepping stone to this new format.  The current implementation does not yet represent the complete architecture described in the link above, but it is far enough along that we are proud of what has been accomplished and look forward to improving it with every release of ManageIQ.
+- VMware VimBroker Removal - In addition to the various performance issues with the VMware VimBroker, one of the underlying technologies, [DRb](https://en.wikipedia.org/wiki/Distributed_Ruby), was incompatible with Podification. As such, we redesigned the VMware interaction in such a way that the broker was removed completely. Much more detail can be found [here](http://talk.manageiq.org/t/vmware-vimbroker-removal/4775)
```
Comment (Fryguy) [inline site/_posts/2020-03-31-Announcing-Jansa-Alpha.md]: Minor...missing period at the end after the `[here]()` link

**Comment by Fryguy** [comment]
I think we need to update the download page to include a link or something to instructions on how to run/download the podified version for kubernetes and openshift.  The only thing on there now is docker.

**Comment by chessbyte** [comment]
Asked @carbonin to add a Getting Started on Kubernetes page to https://www.manageiq.org/docs/get-started/

**Comment by chessbyte** [comment]
Added a link to https://www.manageiq.org/docs/get-started/kubernetes

**Pull request merged**